tests/integration/api_v2/rpc/blocks.test.js

```diff
@@ -43,8 +43,13 @@ describe('Method get.blocks', () => {
 	let refBlock;
 	let refDelegate;
 	beforeAll(async () => {
+		let response;
 		[refBlock] = (await getBlocks({ limit: 1, offset: 2 })).result.data;
-		[refDelegate] = (await request(wsRpcUrl, 'get.accounts', { isDelegate: true, limit: 1 })).result.data;
+		do {
+			// eslint-disable-next-line no-await-in-loop
+			response = await request(wsRpcUrl, 'get.accounts', { isDelegate: true, limit: 1 });
+		} while (!response.result);
```

Comment:
Kindly verify the same as stated above.

Reply:
Here result is either undefined or with data. Thats the reason, I added check for result.